docs/intree-migrate.md

```diff
@@ -0,0 +1,229 @@
+# In-tree storage plugin to CSI Driver Migration
+
+- [Prerequisite](#prerequisite)
+- [Existing in-tree volume operations](#existing-in-tree-volume-operations)
+  - [Mount volume to a POD](#mount-volume-to-a-pod)
+  - [Unmount volume](#unmount-volume)
+  - [Delete volume](#delete-volume)
+- [New Volume operations](#new-volume-operations)
+  - [Create volume](#create-volume)
+  - [Mount volume to a POD](#mount-volume-to-a-pod)
+  - [Resize volume](#resize-volume)
+  - [Unmount volume](#unmount-volume)
+  - [Delete volume](#delete-volume)
+
+## Prerequisite
+
+- For in-tree RBD migration to CSI driver to be supported for your kubernetes
+  cluster, the kubernetes version running in your cluster should be >= v1.23. We
+  also need sidecar controllers ( csi-provisioner and csi-resizer) which are
+  compatible with the kubernetes version v1.23 to be available in your cluster.
+  You can enable the migration with a couple of feature gates in your kubernetes
+  cluster. These feature gates are alpha in kubernetes 1.23 release.
+
+```
+  CSIMigrationRBD: when enabled, it will redirect traffic from in-tree rbd
```

Comment:
why is this mono-spaced? A format like

```markdown
* `CSIMigrationRBD`: when enabled, it will redirect t..
* `IntreePluginRBDUnregister`: Disables the RBD in-tree driver
```

will be easier to read.

Reply:
sure, done @nixpanic